tests/models/test_lora_layers.py

```diff
@@ -921,9 +923,9 @@ def test_sdxl_0_9_lora_three(self):
         ).images
 
         images = images[0, -3:, -3:, -1].flatten()
-        expected = np.array([0.4115, 0.4047, 0.4124, 0.3931, 0.3746, 0.3802, 0.3735, 0.3748, 0.3609])
+        expected = np.array([0.4015, 0.3761, 0.3616, 0.3745, 0.3462, 0.3337, 0.3564, 0.3649, 0.3468])
```

Comment:
Updating the expected values here because I'm quite sure that this PR corrects a bug.
The lora: 

```
        lora_model_id = "hf-internal-testing/sdxl-0.9-kamepan-lora"
        lora_filename = "kame_sdxl_v2-000020-16rank.safetensors"
```

is quite unique since it has different `network_alphas` for q,k,v and out LoRA of the attention layer. Previously we always set the same `network_alpha` for all q,k,v and out (see [here](https://github.com/huggingface/diffusers/blob/3bba44d74eaf20af0d14424d786381f0103c6c90/src/diffusers/models/attention_processor.py#L546))

However, it might very well be that q,k,v and out have different `network_alpha` values. This is now possible as of this PR and therefore should mean it's a bug fix since this checkpoint has different `network_alpha`s for each q,k,v and out.

Reply:
https://github.com/huggingface/diffusers/blob/3bba44d74eaf20af0d14424d786381f0103c6c90/src/diffusers/loaders.py#L1407 suggests we assign different alphas to the modules you mentioned. What am I missing out on?

Reply:
Ah got it. We do that for the text encoder only it seems but not the UNet, is my understanding correct?

Reply:
Yeah exactly! I'm pretty sure it's corrected now :-)